vmm/src/vm.rs

```diff
@@ -241,7 +242,24 @@ impl Vm {
 
         let kernel = File::open(&config.lock().unwrap().kernel.as_ref().unwrap().path)
             .map_err(Error::KernelFile)?;
-        let fd = kvm.create_vm().map_err(Error::VmCreate)?;
+
+        let fd: VmFd;
+        loop {
+            match kvm.create_vm() {
+                Ok(res) => fd = res,
+                Err(e) => {
+                    if e == errno::Error::new(libc::EINTR) {
```

Comment:
A bit of a nit, but I feel something like:

```rust
if e.errno() == libc::EINTR {
}
```

reads better.

Reply:
Oh definitely :)